plugins/ingest/src/main/java/org/elasticsearch/plugin/ingest/PipelineExecutionService.java

```diff
@@ -43,7 +43,7 @@ public PipelineExecutionService(PipelineStore store, ThreadPool threadPool) {
     public void execute(Data data, String pipelineId, Listener listener) {
         Pipeline pipeline = store.get(pipelineId);
         if (pipeline == null) {
-            listener.failed(new IllegalArgumentException(LoggerMessageFormat.format("pipeline with id [{}] does not exist", pipelineId)));
+            listener.failed(new IllegalArgumentException(LoggerMessageFormat.format("pipeline with id [{}] does not exist", new Object[]{pipelineId})));
```

Comment:
I see that we are now doing ` new Object[]{pipelineId}`. I don't get why though, wasn't just `pipelineId` ok?